src/hotspot/share/opto/ifnode.cpp

```diff
@@ -2227,6 +2228,9 @@ void ParsePredicateNode::dump_spec(outputStream* st) const {
     case Deoptimization::DeoptReason::Reason_loop_limit_check:
       st->print("Loop Limit Check ");
       break;
+    case Deoptimization::DeoptReason::Reason_short_running_loop:
+      st->print("Short_Running_Loop ");
```

Comment:
```suggestion
    case Deoptimization::DeoptReason::Reason_short_running_long_loop:
      st->print("Short_Running_Long_Loop ");
```
Might be helpful that it is specifically about a long loop when debugging.